diff --git a/dist/index.js b/dist/index.js
index 9f50c8bc1dcf81ffd5857ce43c3a584dc5f91ada..a8176425d9f7559e2576497962783618b8de69d4 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -80,7 +80,7 @@ function createIntegration(args) {
         const entryUrl = new URL(_buildConfig.serverEntry, _buildConfig.server);
         const pth = fileURLToPath(entryUrl);
         await esbuild.build({
-          target: "es2020",
+          target: "esnext",
           platform: "browser",
           entryPoints: [pth],
           outfile: pth,
